tests/integration/test_recipe.py

```diff
@@ -2165,7 +2165,8 @@ def test_fx_list_mip_change_cmip6(tmp_path, patched_datafinder, config_user):
     assert '_Ofx_' in fx_variables['areacello']['filename']
     assert '_Efx_' in fx_variables['clayfrac']['filename']
     assert '_fx_' in fx_variables['sftlf']['filename']
-    assert '_fx_' in fx_variables['sftgif']['filename']
+    assert len(fx_variables['sftgif']['filename']) == 1
+    assert 'sftgif_' in fx_variables['sftgif']['filename'][0]
```

Comment:
@sloosvel am a bit concerned about this fix I had to introduce, isn't `fx_variables['sftgif']['filename']` a string no matter what fx variable we look at? Why should it be a list for `sftgif` and regular string for the other fx vars?

Reply:
It can be a list if the fx var is time dependent, since therefore the data can be split across different files if it's chunked. And `sftgif` is present in many of the CMOR tables. I think that the difference is that the current implementation look only in the `fx` mip (which is time independent), unless specified by the user. Whereas this one looks at all the tables unless specified. I think during the integration tests it finds an sftgif that belongs to `mip: IyrAnt`, but I don't think that is a problem?

Reply:
OK good to know Saskia, cheers for the clarification, makes good sense. The test is failing on Circle because of the test codebase being used from the release, and not from current development branch, see #1063 - please don't fix it, I need to run a GA test be 100% sure that issue is valid :+1:

Reply:
Actually I think it's an old test, because my local branch does not look like that. In that case sftgif is only a string. But it could also be a list if the file was time dependent. I'll update the branch so this test is fixed.

Reply:
cool, cheers Saskia! note that the issue is still present (not related to your PR, but the issue of older codebase tested with with up-to-date tests in Circle). Best advice is to run tests locally too, everytime one declares a PR ready for merge :+1: